imap_processing/ultra/decom_ultra.py

```diff
@@ -0,0 +1,160 @@
+# Standard
+import logging
+
+import numpy as np
+
+# Installed
+import xarray as xr
+
+# Local
+from imap_processing import decom
+
+logging.basicConfig(level=logging.INFO)
+#TODO: This is a work in progress. Hard-coded values will be removed.
+def read_n_bits(binary: str, n: int, current_position: int):
+    """
+    Extract a specified number of bits from a binary string,
+    starting from a given position.
+
+    Parameters
+    ----------
+    binary : str
+        The binary string from which bits will be read.
```

Comment:
```suggestion
        The string of binary data from which bits will be read.
```

"Binary string" has a special meaning in python `b"test"`, is that what this is? Or is it just a string of 0s and 1s?